test/convenience-method-test.js

```diff
@@ -333,7 +378,7 @@ test("Convenience method", function (t) {
 
   t.test(".rescheduleJob(job, RecurrenceRule)", function(t) {
     t.test("Reschedule jobs from RecurrenceRule to RecurrenceRule", function (test) {
-      test.plan(3);
+      test.plan(4);
```

Comment:
Why did amount of assertions increase here?

Reply:
My bad, was coding late and just did this to get past the test temporarily, left it in (there's obviously still a problem with my implementation as it does not pass with 3